packages/reactotron-app/App/Stores/DevTrackerStore.js

```diff
@@ -0,0 +1,77 @@
+import { computed, observable, action, asMap } from "mobx"
+
+const TIME_NAME = 'devTrackerTimeConnected'
+const ERROR_NAME = 'devTrackerErrorsCount'
+
+class DevTracker {
+    @observable storedTime = 0
+    @observable currentErrors = 0
+
+    @observable sessionStarted = null
+
+    constructor() {
+        const currentDevTime = localStorage.getItem(TIME_NAME)
+        const currentErrors = localStorage.getItem(ERROR_NAME)
+
+        if (typeof currentDevTime === 'undefined' || currentDevTime === null) {
+            localStorage.setItem(TIME_NAME, 0)
+        } else {
+            this.storedTime = parseInt(currentDevTime, 10)
+        }
```

Comment:
If we have a value in local storage use it, if not lets set it to 0 just so we have SOMETHING.